templates/components/preview.html

```diff
@@ -77,7 +77,7 @@
   ></p>
   <div class="preview-wrapper">
     <div class="preview-img">
-      <img id="preview-img" src="../assets/thumbnail.svg" />
+      <img id="preview-img" src="../assets/thumbnail.svg'" />
```

Comment:
I'm not sure about this `'` character in path ...

Reply:
Good catch